mongo/integration/change_stream_test.go

```diff
@@ -236,6 +236,7 @@ func TestChangeStream_ReplicaSet(t *testing.T) {
 			Code:    errorHostUnreachable,
 			Name:    "foo",
 			Message: "bar",
+			Labels:  []string{"ResumableChangeStreamError"},
```

Comment:
How is this related to other changes?

Reply:
I switched the mock deployment to use the latest wire version we support (9, or server version 4.4), which was an upgrade from the existing reported version of 8 (or server version 4.2). For server versions >= 4.4, change streams consider a server error to be resumable if it has the `ResumableChangeStreamError` label, so this test needed to be modified to include this label.